centos_ci.sh

```diff
@@ -105,6 +115,10 @@ make cross
 crc setup
 make integration
 
+if [[ $? -ne 0 ]]; then
+	perform_test_logs_upload $CICO_API_KEY
```

Comment:
you then need to yourself do `exit 1` since you are already capturing the last exit code so if something fails your logs uploaded to artifact location but it will be then successful as job.

Reply:
Right..